### ISSUE TITLE
cannot open links in Custom Menu in new window

### ISSUE THREAD

**Issue by suzannezc** [wrdsb/wordpress-theme#192]

**Comment by suzannezc** [comment]
https://wordpress.org/support/topic/custom-menu-open-link-in-a-new-window
